WalletWasabi.Fluent/ViewModels/HomePageViewModel.cs

```diff
@@ -1,14 +1,34 @@
 using ReactiveUI;
+using System.Collections.ObjectModel;
+using System.Reactive.Linq;
+using DynamicData;
+using DynamicData.Binding;
 
 namespace WalletWasabi.Fluent.ViewModels
 {
 	public class HomePageViewModel : NavBarItemViewModel
 	{
-		public HomePageViewModel(IScreen screen) : base(screen)
+		private readonly ReadOnlyObservableCollection<NavBarItemViewModel> _items;
+
+		public HomePageViewModel(IScreen screen, WalletManagerViewModel walletManager, AddWalletPageViewModel addWalletPage) : base(screen)
 		{
 			Title = "Home";
+
+			var list = new SourceList<NavBarItemViewModel>();
+			list.Add(addWalletPage);
+
+			walletManager.Items.ToObservableChangeSet()
+				.Cast(x => x as NavBarItemViewModel)
+				.Sort(SortExpressionComparer<NavBarItemViewModel>.Ascending(i=>i.Title))
+				.Or(list.Connect())
+				.Sort(SortExpressionComparer<NavBarItemViewModel>.Ascending(i => i is AddWalletPageViewModel ? 1 : 0))
```

Comment:
Merge would work here, otherwise we get wrong order of wallets
```C#
			walletManager.Items.ToObservableChangeSet()
				.Cast(x => x as NavBarItemViewModel)
				.Sort(SortExpressionComparer<NavBarItemViewModel>.Ascending(i=>i.Title))
				.Merge(list.Connect())
				//.Sort(SortExpressionComparer<NavBarItemViewModel>.Ascending(i => i is AddWalletPageViewModel ? 1 : 0))
				.ObserveOn(RxApp.MainThreadScheduler)
				.Bind(out _items)
				.AsObservableList();
```